src/Drupal/Commands/field/FieldDefinitionCommands.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drush\Drupal\Commands\field;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Field\FieldTypePluginManagerInterface;
+use Drupal\Core\Field\FormatterPluginManager;
+use Drupal\Core\Field\WidgetPluginManager;
+use Drush\Attributes as CLI;
+use Drush\Commands\DrushCommands;
+
+/**
+ * Defines field definition Drush commands.
+ */
+final class FieldDefinitionCommands extends DrushCommands {
+
+    public function __construct(
+        private readonly FieldTypePluginManagerInterface $typePluginManager,
```

Comment:
Love the promoted properties. I used there on our Attribute classes - good idea to use in commandfiles.